cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/kubernetes/AbstractNamespacedResourceOperator.java

```diff
@@ -403,33 +404,33 @@ public Future<Void> deleteAsync(Reconciliation reconciliation, String namespace,
-     * informer returned by this method is not running and has to be started by the code using it.
-     *
-     * @param namespace         Namespace on which to inform
-     * @param resyncIntervalMs  The interval in which the resync of the informer should happen in milliseconds
-     *
-     * @return          Informer instance
-     */
-    public SharedIndexInformer<T> informer(String namespace, long resyncIntervalMs)   {
-        return runnableInformer(applyNamespace(namespace), resyncIntervalMs);
-    }
-
-    /**
-     * Creates the informer for given resource type to inform on all instances in given namespace (or cluster-wide)
-     * matching the selector. The informer returned by this method is not running and has to be started by the code
-     * using it.
-     *
-     * @param namespace         Namespace on which to inform
-     * @param selectorLabels    Selector which should be matched by the resources
-     * @param resyncIntervalMs  The interval in which the resync of the informer should happen in milliseconds
-     *
-     * @return                  Informer instance
-     */
-    public SharedIndexInformer<T> informer(String namespace, Map<String, String> selectorLabels, long resyncIntervalMs)   {
-        return runnableInformer(applyNamespace(namespace).withLabels(selectorLabels), resyncIntervalMs);
-    }
+    ///**
+    // * Creates the informer for given resource type to inform on all instances in given namespace (or cluster-wide). The
+    // * informer returned by this method is not running and has to be started by the code using it.
+    // *
+    // * @param namespace         Namespace on which to inform
+    // * @param resyncIntervalMs  The interval in which the resync of the informer should happen in milliseconds
+    // *
+    // * @return          Informer instance
+    // */
+    //public Informer<T> informer(String namespace, long resyncIntervalMs)   {
+    //    return new Informer<>(runnableInformer(applyNamespace(namespace), resyncIntervalMs));
+    //}
+    //
+    ///**
+    // * Creates the informer for given resource type to inform on all instances in given namespace (or cluster-wide)
+    // * matching the selector. The informer returned by this method is not running and has to be started by the code
+    // * using it.
+    // *
+    // * @param namespace         Namespace on which to inform
+    // * @param selectorLabels    Selector which should be matched by the resources
+    // * @param resyncIntervalMs  The interval in which the resync of the informer should happen in milliseconds
+    // *
+    // * @return                  Informer instance
+    // */
+    //public Informer<T> informer(String namespace, Map<String, String> selectorLabels, long resyncIntervalMs)   {
+    //    return new Informer<>(runnableInformer(applyNamespace(namespace).withLabels(selectorLabels), resyncIntervalMs));
+    //}
```

Comment:
can we remove all the above commented code?

Reply:
Damn, I thought I deleted all of them. Thanks for catching this.